chain/gen/mining.go

```diff
@@ -34,10 +32,10 @@ func MinerCreateBlock(ctx context.Context, sm *stmgr.StateManager, w *wallet.Wal
 	next := &types.BlockHeader{
 		Miner:                 miner,
 		Parents:               parents.Cids(),
-		Tickets:               tickets,
+		Ticket:                ticket,
```

Comment:
from https://github.com/filecoin-project/lotus/pull/666/files#diff-8e5a85767f3dfe2cb51b9529121aa162R475, I think you're doing it right but I don't like the name `Ticket`. It should be an array of all winning EPoSt Partial Tickets